mainapp/views.py

```diff
@@ -9,3 +9,11 @@ def index(request):
 @login_required
 def account(request):
     return render(request, "account.html", {"user": request.user})
+
+
+@login_required
+def favlist(request):
+    favorited_apartments = request.user.favorites.all()
+    return render(
+        request, "favlist.html", {"favorited_apartments": favorited_apartments}
+    )
```

Comment:
per my previous comment, I think this makes more sense as part of the location package than the mainapp package.

Reply:
Done!